pkg/apis/monitoring/v1alpha1/types.go

```diff
@@ -184,6 +184,10 @@ type PrometheusConfig struct {
 	// Enable Prometheus to be used as a receiver for the Prometheus remote write protocol. Defaults to the value of `false`.
 	// +optional
 	EnableRemoteWriteReceiver bool `json:"enableRemoteWriteReceiver,omitempty"`
+	// Enable Prometheus to accept OpenTelemetry Metrics via the otlp/http protocol.
+	// Defaults to the value of `false`.
+	// +optional
+	EnableOtlpHttpReceiver bool `json:"enableOtlpHttpReceiver,omitempty"`
```

Comment:
(suggestion) I'd make it a bool pointer so we can decide later to have it on by default (e.g. opt-in -> opt-out) if needed.
```suggestion
        // +optional
	EnableOtlpHttpReceiver *bool `json:"enableOtlpHttpReceiver,omitempty"`
```

Reply:
But it is already `+optional`? :)